### PR TITLE
Server and DB Setup

### PR DESCRIPTION
**Pull request by emkwak**

### CODE REVIEW

File: database/models/Event.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 const mongoose = require('mongoose');
 const db = require('../index.js');
-const Member = require('./Member.js')
+const Member = require('./Member.js').schema;
```
Comment (WDHudson) [inline database/models/Event.js]: Interesting. I've never seen it done this way before. I assume this works just fine if you already populated your DB.

File: database/seed.js [hunk 2]
```diff
@@ -48,3 +51,26 @@ for (let i = 0; i < 100; i++) {
 
   events.push(newEvents);
 }
+
+const insertSampleMembersAndEvents = function () {
+  Member.deleteMany((err) => {
+    console.log('Removed Member')
+  })
+    .then(() =>
+      Event.deleteMany((err) => {
```
Comment (WDHudson) [inline database/seed.js]: If you're using promises, are you using any libraries such as Bluebird? If so, then doesn't that need to be required at the top of the file?

File: database/models/Event.js (hunk 1 above)
Comment (emkwak) [reply database/models/Event.js]: @WDHudson I was getting an error on the terminal when I ran the seed script and I googled it. If I wanted to use another model in my schema I needed to add .schema at the end

File: database/seed.js (hunk 2 above)
Comment (emkwak) [reply database/seed.js]: @WDHudson I'm just using JS Promises that doesn't require anything

File: database/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+var mongoose = require('mongoose');
+const mongoUri = 'mongodb://localhost/attendees';
+
+mongoose.connect(mongoUri, { useNewUrlParser: true });
```
Comment (manuelag19) [inline database/index.js]: Nice, I was getting a deprecation error about this parser

File: database/seed.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    attendees,
+    eventOrganizer,
+    waitlist,
+  };
+
+  events.push(newEvents);
+}
+
+const insertSampleEvents = function () {
+  Event.deleteMany((err) => {
```
Comment (manuelag19) [inline database/seed.js]: Are you clearing your database every time your seed script runs?

Comment (emkwak) [reply database/seed.js]: @manuelag19 yes, so it doesn't save more than 100 Events to the DB every time I run the seed script. This was mainly for when I testing my seed script, but I thought it was fine to leave it

**Pull request merged**